Assessment.Test/Assessment.Test/Services/Interfaces/IHttpRequestProviderService.cs

```diff
@@ -16,6 +18,6 @@ public interface IHttpRequestProviderService
         /// <param name="content"></param>
         /// <returns></returns>
         Task<HttpResponseMessage> PostAsync(string url, object content);
-
+        Task<STRootModel> GetAsync();
```

Comment:
Use this method as a generic method. Coz this service is need be a common service for all API transactions.
Ex - public async Task<T> GetAsync<T>()